### PR TITLE
Assignment 4 from Adrian Rejas Llamera

### PR DESCRIPTION
**Pull request by AdriRejas**
I could not find my ID student.

### CODE REVIEW

File: Assignment4/Assignment 4_AdriRejas/task07.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+for s, p, o in g.triples((None, RDFS.subClassOf, ns.LivingThing)):
+  print(s)
+
+# SPARQL
+print("SPARQL queries")
+q1 = prepareQuery('''
+    SELECT ?subclass
+    WHERE {
+        ?subclass rdfs:subClassOf ns:LivingThing .
+    }
```
Comment (anaigmo) [inline Assignment4/Assignment 4_AdriRejas/task07.py:47]: missing recursive subclassof relationships

File: Assignment4/Assignment 4_AdriRejas/task07.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+# TO DO
+q6 = prepareQuery('''
+    SELECT ?entity  WHERE {
+        ?entity foaf:knows ?person1 .
+        ?entity foaf:knows ?person2 .
+        ?entity vcard:Given ?Name.
+        FILTER (?person1 != ?person2)
+    }
+    GROUP BY ?entity
+    HAVING (COUNT(?person1) >= 2)
```
Comment (anaigmo) [inline Assignment4/Assignment 4_AdriRejas/task07.py:166]: redundant with filter

File: Assignment4/Assignment 4_AdriRejas/task07.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+            vcard:Given ?Given.
+	?RockySmith vcard:FN ?RockySmithFullName .
+  }
+  ''',
+  initNs = { "foaf": FOAF, "vcard": VCARD, "xsd":XSD}
+)
+for r in g.query(q5, initBindings = {'?RockySmithFullName' : Literal('Rocky Smith', datatype=XSD.string)}):
+  print(r.Given)
+
+"""**Task 7.5: List the entities who know at least two other entities in the graph**"""
```
Comment (anaigmo) [inline Assignment4/Assignment 4_AdriRejas/task07.py:155]: missing rdflib alternative in this exercise

**Comment by AdriRejas** [comment]
I already updated all the changes on tasks 6 and 7. I dont know if changes are correctly sent. Please let me know.

**Review by AdriRejas — COMMENTED** [review]
tasks 06 and 07 updated

**Pull request merged**